docs/ml-clustering.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: global
+title: Clustering - ML
+displayTitle: <a href="ml-guide.html">ML</a> - Clustering
+---
+
+In `spark.ml`, we implement the corresponding pipeline API for 
+[clustering in mllib](mllib-clustering.html).
+
+## Latent Dirichlet allocation (LDA)
+
+`LDA` is implemented as an Estimator that supports both `EMLDAOptimizer` and `OnlineLDAOptimizer`,
+and generates `LocalLDAModel` and `DistributedLDAModel` respectively, as the base models.
```

Comment:
This isn't true; `LDA` generates a `LDAModel` which is the superclass of wrappers to the two `mllib` classes. I would just say "generates a `LDAModel`".

We can also mention that expert users may cast a `LDAModel` generated by `EMLDAOptimizer` to a `DistributedLDAModel` if needed.